frontend/src/components/page/ContestTop/ContestTop.tsx

```diff
@@ -1,3 +1,78 @@
+import { useGetContest, useRegisterMe, useRouterContestSlug } from "@/src/usecases/contest";
+import { Box, Button, Card, CardBody, CardHeader, Heading, useToast } from "@chakra-ui/react";
+import { useTimer } from "react-timer-hook";
+
+const TimerComponent = (props: { expiryTimestamp: Date, timerHeader: string}) => {
+  const onExpire = () => window.location.reload();
+  const timer = useTimer({ expiryTimestamp: props.expiryTimestamp, onExpire });
+  // leading zero を追加する処理
+  const seq = [
+    timer.days.toLocaleString(),
+    timer.hours.toLocaleString(),
+    timer.minutes.toLocaleString(),
+    timer.seconds.toLocaleString(),
+  ];
+  for (let i = 0; i < 4; i++) {
+    if (seq[i].length == 1) seq[i] = "0" + seq[i];
+  }
+  return <div>
+    {props.timerHeader + seq[0] + ":" + seq[1] + ":" + seq[2] + ":" + seq[3]}
+  </div>
```

Comment:
超細かいのですが，修正お願いします:bow:

```suggestion
  return <span>
    {props.timerHeader + seq[0] + ":" + seq[1] + ":" + seq[2] + ":" + seq[3]}
  </span>
```
ContestTop 側が `<p>` の中で TimerComponent を呼び出しているので `<div>` だと `<p><div></div></p>` になって HTML として正しくないので div 以外のタグに変更をお願いします．

![image](https://github.com/szpp-dev-team/szpp-judge/assets/61489178/25221af1-19d7-4d49-b5bd-19eb1ff64117)

Reply:
span タグに変更しました